tests/unit/test_logging.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import platform
+
+from django.conf import settings
+from django.test import override_settings
+
+from aap_eda.utils import get_eda_version
+from aap_eda.utils.logging import SETTINGS_LIST_FOR_LOGGING, startup_logging
+
+
+@override_settings(SECRET_KEY="some_random_not_so_secret_key")
+def test_startup_logging(caplog_factory):
+    logger = logging.getLogger(__name__)
+    caplog = caplog_factory(logger)
+    startup_logging(logger)
+
+    assert "Starting eda-server" in caplog.text
+    assert get_eda_version() in caplog.text
+    assert "Python version" in caplog.text
+    assert platform.python_version() in caplog.text
+    for setting in SETTINGS_LIST_FOR_LOGGING:
+        assert setting in caplog.text
+        assert str(getattr(settings, setting)) in caplog.text
+
+    # some assertions to check that sensitive data is not logged
+    assert "some_random_not_so_secret_key" not in caplog.text
```

Comment:
Minor nit - 
I think you could also use
```
assert settings.SECRET_KEY not in caplog.text
```
Instead of passing in this string directly

Reply:
Agree, nice suggestion. I will be honest. I'm scared to lose the approvals, it is hard to get them xD

Reply:
Next iteration perhaps :)

Reply:
Nope, I just was crying. I don't like to leave known issues, specially when they are so trivial. :)
It is updated, please review again.